game-authorization-score/src/main/java/com/iconbet/score/authorization/Authorization.java

```diff
@@ -0,0 +1,965 @@
+    private static final String SUPER_ADMIN = "super_admin";
+    private static final String PROPOSAL_DATA = "proposal_data";
+    private static final String PROPOSAL_LIST = "proposal_list";
+    private static final String STATUS_DATA = "status_data";
+    private static final String OWNER_DATA = "owner_data";
+    private static final String ROULETTE_SCORE = "roulette_score";
+    private static final String DAY = "day";
+    private static final String PAYOUTS = "payouts";
+    private static final String WAGERS = "wagers";
+    private static final String NEW_DIV_CHANGING_TIME = "new_div_changing_time";
+    private static final String GAME_DEVELOPERS_SHARE = "game_developers_share";
+    
+    private static final String TODAYS_GAMES_EXCESS = "todays_games_excess";
+    // dividends paid according to this excess
+    private static final String GAMES_EXCESS_HISTORY = "games_excess_history";
+    		
+
+    private static final String APPLY_WATCH_DOG_METHOD = "apply_watch_dog_method";
+    private static final String MAXIMUM_PAYOUTS = "maximum_payouts";
+    private static final String MAXIMUM_LOSS = "maximum_loss";
+        
+
+	private final VarDB<Address> roulette_score = Context.newVarDB(ROULETTE_SCORE, Address.class);
+	private final ArrayDB<Address> admin_list = Context.newArrayDB(ADMIN_LIST, Address.class);
+	private final VarDB<Address> super_admin = Context.newVarDB(SUPER_ADMIN, Address.class);
+	//  question=?  why adrress
+	private final DictDB<Address,String> proposal_data = Context.newDictDB(PROPOSAL_DATA, String.class);
+	private final DictDB<Address,String> status_data = Context.newDictDB(STATUS_DATA, String.class);
+	private final DictDB<Address,Address> owner_data = Context.newDictDB(OWNER_DATA, Address.class);
+	private final ArrayDB<Address> proposal_list = Context.newArrayDB(PROPOSAL_LIST, Address.class);
+	private final VarDB<BigInteger> day = Context.newVarDB(DAY, BigInteger.class);
+	private final DictDB<Address, BigInteger[]> wagers = Context.newDictDB(WAGERS, BigInteger[].class);
+	private final DictDB<Address, BigInteger[]> payouts = Context.newDictDB(PAYOUTS, BigInteger[].class);
+	
+	private final VarDB<BigInteger> game_developers_share = Context.newVarDB(GAME_DEVELOPERS_SHARE, BigInteger.class);
+	private final DictDB<Address,BigInteger> todays_games_excess = Context.newDictDB(TODAYS_GAMES_EXCESS, BigInteger.class);
+	
+	private final VarDB<BigInteger> new_div_changing_time = Context.newVarDB(NEW_DIV_CHANGING_TIME, BigInteger.class);
+	private final DictDB<Address, BigInteger[]> games_excess_history = Context.newDictDB(TODAYS_GAMES_EXCESS, BigInteger[].class);
+
+	private final VarDB<Boolean> apply_watch_dog_method = Context.newVarDB(APPLY_WATCH_DOG_METHOD, Boolean.class);
+	private final DictDB<Address,BigInteger> maximum_payouts = Context.newDictDB(MAXIMUM_PAYOUTS, BigInteger.class);
+	private final VarDB<BigInteger> maximum_loss = Context.newVarDB(MAXIMUM_LOSS, BigInteger.class);
+
+	  
+
+	@Override
```

Comment:
remove the methods inherited from TokenStandard IRC2

Reply:
removed.